aws/resource_aws_autoscaling_group.go

```diff
@@ -834,7 +838,7 @@ func resourceAwsAutoscalingGroupDrain(d *schema.ResourceData, meta interface{})
 
 	// Next, wait for the autoscale group to drain
 	log.Printf("[DEBUG] Waiting for group to have zero instances")
-	return resource.Retry(10*time.Minute, func() *resource.RetryError {
+	return resource.Retry(d.Timeout(schema.TimeoutDelete), func() *resource.RetryError {
```

Comment:
Ah no, that's a strange typo or copy/paste error, I'll bump that back up to 10, thanks!